docs/graphql/manual/actions/codegen/examples/index.rst

```diff
@@ -0,0 +1,29 @@
+.. meta::
+   :description: Examples for Hasura codegen
+   :keywords: hasura, docs, actions, codegen, examples
+
+.. _actions_codegen_examples:
+
+Codegen examples
+================
+
+.. contents:: Table of contents
+  :backlinks: none
+  :depth: 2
+  :local:
+
+Introduction
+------------
+
+As of now, Hasura provides codegen for a few frameworks (``nodejs-express``,
```

Comment:
nodejs-express and typescript-zeit are not mentioned in examples though?

Reply:
@praveenweb no we have only python-flask for now. @GavinRay97 will add more in the future. I now re-wrote the section a little so that it's less confusing. Please check it out.